### ISSUE TITLE
Desktop client exe doesn't have icon

### ISSUE THREAD

**Issue by gumbyscout** [nictuku/stardew-rocks#38]
Currently, the executable itself doesn't provide an icon. I see two possible solutions
- Bundle it with the .exe like most windows programs
- Include a icon with the .exe on releases

I say the first option is best option, since the user doesn't have to manually set the icon. I'm not sure how you guys will implement it, but [here](http://stackoverflow.com/questions/25602600/how-do-you-set-the-application-icon-in-golang) is a stackoverflow question I found in relation to it.

**Comment by nictuku** [comment]
I don't think this needs to use a Go-specific solution. The one mentioned in the article (rsrc) has architecture-specific code and currently may not work for amd64.

So I'll probably use something like this instead:
https://github.com/atom/rcedit

Which I found thanks to this question:
http://stackoverflow.com/questions/673203/add-icon-to-existing-exe-file-from-the-command-line
